docs/basics/101-116-sharelocal.rst

```diff
@@ -263,40 +265,94 @@ explore the file hierarchy. The file content, however, is not present yet.
 
 What has happened here?
 
-When DataLad installs a dataset, it will by default only install the
+When DataLad installs a dataset, it will by default only obtain the
 superdataset, and not any subdatasets. The superdataset contains the
 information that a subdataset exists though -- the subdataset is *registered*
 in the superdataset.  This is why the subdataset name exists as a directory.
-A subsequent :command:`datalad install` in ``recordings/longnow/``
-or a :command:`datalad install PATH/TO/longnow` will install the registered dataset without
-the need to specify the source again, just as we did it in the example above.
+A subsequent :command:`datalad get -n path/to/longnow` will install the registered
+subdataset again, just as we did it in the example above.
+
+.. findoutmore:: More on datalad get
+
+   Previously, we used :command:`datalad get` to get file content. However,
+   :command:`get` can operate on more than just the level of *files* or *directories*.
+   Instead, it can also operate on the level of *datasets*. Regardless of whether
+   it is a single file (such as ``books/TLCL.pdf``) or a registered subdataset
+   (such as ``recordings/longnow``), :command:`get` will operate on it to 1) install
+   it -- if it is a not yet installed subdataset -- and 2) retrieve the contents of any files.
+   That makes it very easy to get your file content, regardless of
+   how your dataset may be structured -- it is always the same command, and DataLad
+   blurs the boundaries between superdatasets and subdatasets.
+
+   In the above example, we called :command:`datalad get` with the option ``-n/--no-data``.
+   This option prevents that :command:`get` obtains the data of individual files or
+   directories, thus limiting its scope to the level of datasets as only a
+   :command:`datalad clone` is performed. Without this option, the command would
+   have retrieved all of the subdatasets contents right away. But with ``-n/--no-data``,
+   it only installed the subdataset to retrieve the meta data about file availability.
+
+To explicitly install all potential subdatasets *recursively*, that is,
+all of the subdatasets inside it as well, one can give the
+``-r``/``--recursive`` option to :command:`get`::
+
+  datalad get -n -r <subds>
+
+This would install the ``subds`` subdataset and all potential further
+subdatasets inside of it, and the meta data about file hierarchies would
+have been available right away for every subdataset inside of ``subds``.
+
+So why is a recursive get not the default behavior?
+In :ref:`nesting` we learned that datasets can be nested *arbitrarily* deep.
+Upon getting the meta data of one dataset you might not want to also install
+a few dozen levels of nested subdatasets right away.
```

Comment:
Filed https://github.com/datalad-handbook/book/issues/327 regarding this aspect.

Reply:
Tried with b2a1cf1